@here/harp-mapview-decoder/lib/TileDataSource.ts

```diff
@@ -163,6 +163,10 @@ export class TileDataSource<TileType extends Tile> extends DataSource {
                     `concurrentDecoderServiceName`
             );
         }
+        this.m_copyrightProvider =
+            m_options.copyrightProvider !== undefined
+                ? m_options.copyrightProvider
+                : new DefaultCopyrightProvider();
```

Comment:
If there is no copyright provider given, we should not display any copyright. It would be weird behaviour if the user instantiates a datasource with OSM data and get's HERE copyright